datatype.py

```diff
@@ -0,0 +1,11 @@
+# file  : datatype.py
+#brief  : Get the input from user and convert it into different data types and print it
+#authore: deepti
+#date   : 16-may-17
```

Comment:
Dont use date inside the file, since the date of your changes is managed by git itself